### ISSUE TITLE
zsh/mac problem, can't see .env file in current folder

### ISSUE THREAD

**Issue by matthewdeanmartin** [Zaloog/dotenvhub#3]
Repro steps

1. buy an expensive mac.

```bash
pipx install donenvhub
mkdir folder
cd folder
touch .env
dot
```

And I get an empty "Select your .env File" panel and a red "No File Selected" panel

**Comment by Zaloog** [comment]
Hi @matthewdeanmartin,
the way dotenvhub works is, that you create the file in dotenvhub, which would then store the file under your XDG user_data_dir, which should be `~/Library/Application Support/dotenvhub/env_files/<DIRNAME>/<FILENAME>` on an expensive mac.

Then you can use `dotenvhub` to create a copy of that file in your current working directory. With the `Export File to current dir`-Button and also choose a specific name (default is `.env`).

**Comment by matthewdeanmartin** [comment]
Is that implemented? I looked thru the source code and don't see it. 

After running the app,

```
❯ ls ~/Library/Application\ Support/dotenvhub/env_files
(nothing)
```

If I create the file where you suggested, I can get it to show up.

```bash
mkdir ~/Library/Application\ Support/dotenvhub/env_files/some_app
touch ~/Library/Application\ Support/dotenvhub/env_files/some_app/file
```
Now file will show up.

Naming the file `.env` or `file.txt` will cause `dot` to blow up on illegal control name.

**Comment by Zaloog** [comment]
you can manage everything from within the app.

1. use `dot` to open the app
2. press the `New Env File`-Button
3. Enter your content in the TextArea
4. press the `Save Env File` Button. Then a popup appears where you can enter just a \<filename> or a <dirname/filename> (no spaces allowed currently)
5. when confirming, the file will be automatically created at the desired location

**Comment by Zaloog** [comment]
I must admit, some of the controls I would implement differently now that I know the framework better.
So if you think something could be more intuitiv, I am open for suggestions.
Id also add a gif soon showing the basic functionality

**Comment by Zaloog** [comment]
ill close this, feel free to reopen, if issues still persist.
I also released v0.3.0 which should offer a more intuitive Interface
